ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisRootPage.java

```diff
@@ -15,7 +15,7 @@
     }
 
     private static final class Element {
-        public static final By ROOT = By.id("root");
+        public static final By ROOT = By.tagName("body");
```

Comment:
The same issue, Apicurio doesn't use root id, but we might want to use some of our steps for apicurio as well.